manage.py

```diff
@@ -37,6 +37,7 @@
 manager.command(legal_docs.load_advisory_opinions_into_s3)
 manager.command(legal_docs.load_archived_murs)
 manager.command(legal_docs.load_current_murs)
+manager.command(legal_docs.load_current_adrs)
```

Comment:
It should not be called _current_. That adjective is used only for MURs to distinguish _current_ MURs from _archived_ MURs. I am not sure that a similar notion exists for ADRs.